client/header/activity-panel/panels/help.js

```diff
@@ -123,53 +101,53 @@ function getPaymentsItems( props ) {
 			link:
 				'https://docs.woocommerce.com/document/premium-payment-gateway-extensions/?utm_source=help_panel',
 		},
-		showWCPay && {
+		paymentGatewaySuggestions.woocommerce_payments && {
 			title: __(
 				'WooCommerce Payments Start Up Guide',
 				'woocommerce-admin'
 			),
 			link:
 				'https://docs.woocommerce.com/document/payments//?utm_source=help_panel',
 		},
-		showWCPay && {
+		paymentGatewaySuggestions.woocommerce_payments && {
 			title: __( 'WooCommerce Payments FAQs', 'woocommerce-admin' ),
 			link:
 				'https://docs.woocommerce.com/documentation/woocommerce-payments/woocommerce-payments-faqs/?utm_source=help_panel',
 		},
-		showStripe && {
+		paymentGatewaySuggestions.stripe && {
 			title: __( 'Stripe Setup and Configuration', 'woocommerce-admin' ),
 			link:
 				'https://docs.woocommerce.com/document/stripe/?utm_source=help_panel',
 		},
-		showPayPal && {
+		paymentGatewaySuggestions[ 'ppcp-gateway' ] && {
 			title: __(
 				'PayPal Checkout Setup and Configuration',
 				'woocommerce-admin'
 			),
 			link:
 				'https://docs.woocommerce.com/document/2-0/woocommerce-paypal-payments/#section-3',
 		},
-		showSquare && {
+		paymentGatewaySuggestions.square_credit_card && {
 			title: __( 'Square - Get started', 'woocommerce-admin' ),
 			link:
 				'https://docs.woocommerce.com/document/woocommerce-square/?utm_source=help_panel',
 		},
-		showKlarnaCheckout && {
+		paymentGatewaySuggestions.kco && {
 			title: __( 'Klarna - Introduction', 'woocommerce-admin' ),
 			link:
 				'https://docs.woocommerce.com/document/klarna-checkout/?utm_source=help_panel',
 		},
-		showKlarnaPayments && {
+		paymentGatewaySuggestions.klarna_payments && {
 			title: __( 'Klarna - Introduction', 'woocommerce-admin' ),
 			link:
 				'https://docs.woocommerce.com/document/klarna-payments/?utm_source=help_panel',
 		},
-		showPayFast && {
+		paymentGatewaySuggestions.payfast && {
 			title: __( 'PayFast Setup and Configuration', 'woocommerce-admin' ),
 			link:
 				'https://docs.woocommerce.com/document/payfast-payment-gateway/?utm_source=help_panel',
 		},
-		showEway && {
+		paymentGatewaySuggestions.eway_payments && {
```

Comment:
I assume this should be changed to `eway` in relation to this PR: https://github.com/woocommerce/woocommerce-admin/pull/7228

Reply:
Good catch!  Updated.